va/va.h

```diff
@@ -1273,6 +1328,32 @@ typedef union _VAConfigAttribValContextPriority{
     uint32_t value;
 }VAConfigAttribValContextPriority;
 
+/** @name Attribute values for VAConfigAttribProtectedContentCipherAlgorithm */
+/** \brief AES cipher */
+#define VA_PC_CIPHER_AES                    0x00000001
+
+/** @name Attribute values for VAConfigAttribProtectedContentCipherBlockSize */
+/** \brief 128 bits block size */
+#define VA_PC_BLOCK_SIZE_128                0x00000001
+
+/** @name Attribute values for VAConfigAttribProtectedContentCipherMode */
+/** \brief AES CTR */
+#define VA_PC_CIPHER_MODE_CTR               0x00000001
```

Comment:
I did not get question. 
Are you saying someone could OR CTR | CBC? If yes then this will be **huge** mistake from bitstream creators. 
Again, vendors follow CENC v2/v3 standards for video bitstream encryption.

Reply:
config attributes always has two direction, for the query function, it will query ability of HW,  my question is , how to report hw support multiple mode/type ...

Reply:
When call vaGetConfigAttributes with these config attribute, the return values is ORed. Therefore application can check each feature (bit) is supported or not.